documentation/sphinx/source/api-ruby.rst

```diff
@@ -104,6 +104,10 @@ After requiring the ``FDB`` gem and selecting an API version, you probably want
 
         |option-trace-roll-size-blurb|
 
+    .. method:: FDB.options.set_trace_format(format) -> nil
+
+        |option-trace-format-blurb|
+
```

Comment:
Looks like something weird happened here, but it's not from this PR. Not something to fix now, but I just wanted to make a note of it.

Reply:
https://github.com/apple/foundationdb/pull/1132